TorqueDataConnect/include/specials/TorqueDataConnectColumns.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * Special page that lists available spreadsheet columns
+ */
+
+class TorqueDataConnectColumns extends SpecialPage {
+  public function __construct() {
+    parent::__construct('TorqueDataConnectColumns', 'torquedataconnect-admin');
+  }
+
+  public function execute($subPage) {
+    global $wgTorqueDataConnectGroup;
+    global $wgTorqueDataConnectWikiKey;
+    global $wgTorqueDataConnectSheetName;
+
+    $this->setHeaders();
+    $this->checkPermissions();
+    $ch = curl_init();
+    curl_setopt(
+      $ch,
+      CURLOPT_URL,
+      'http:/localhost:5000/api/' .
+      $wgTorqueDataConnectSheetName . '.json' .
+      '?group=' . $wgTorqueDataConnectGroup .
```

Comment:
Because this is only going to return the columns for which this group is configured to have since you're passing in the `$wgTorqueDataConfigGroup`, it's effectively the same list as the configuration for this group.  This issue is about giving admins a list of all columns, regardless of what is currently set up for their group in the MainConfig.  The main thing we want to solve here is when columns get added to the spreadsheet, we want a place for admins to see them.

Or, even better, a way for them to set up the MainConfig to point to this special page so they just always have all columns (this is a good use case for testing this).